### DISCUSSION TITLE
v2.0.0

### DISCUSSION THREAD

**Discussion by caring-coder** [verronpro/office-stamper#360] [Announcements]
## What's Changed
* refactor Maven structure to multi-module project by @caring-coder in https://github.com/verronpro/docx-stamper/pull/349
* BREAKING CHANGES: finalize modularization in 2.0 by @caring-coder in https://github.com/verronpro/docx-stamper/pull/312

**Full Changelog**: https://github.com/verronpro/docx-stamper/compare/v1.6.9...v2.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/verronpro/docx-stamper/releases/tag/v2.0.0'>v2.0.0</a>.</em>

**Comment by StefanLobbenmeierObjego** [comment]
Just wondering how to use this release, is it intended that only engine has been deployed?

I can see engine here: https://mvnrepository.com/artifact/pro.verron.office-stamper

But the parent pom is not released, so its not usable:

```
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not resolve pro.verron.office-stamper:engine:2.0.
     Required by:
         project :
      > Could not resolve pro.verron.office-stamper:engine:2.0.
         > Could not parse POM https://repo.maven.apache.org/maven2/pro/verron/office-stamper/engine/2.0/engine-2.0.pom
            > Could not find pro.verron.office-stamper:office-stamper:2.0.
```

**Reply by caring-coder** [reply]
Indeed i want to deploy it separately, but the dependency on the parent pom is a mistake on my part.
The office-stamper engine is equivalent to the docx-stamper lib, with the core package not exported anymore.
The office-stamper examples is intended to illustrate how to use it, and provide some black box integration tests.

**Reply by caring-coder** [reply]
2.0.1 has been published to fix the dependency issue

**Reply by StefanLobbenmeierObjego** [reply]
Thanks, I could confirm that I can resolve the dependency and all my tests are still passing :smile:

**Comment by caring-coder** [comment]
v2.0.1 is released, fixing the current version issue #385
